data-prepper-plugins/opensearch/src/main/java/com/amazon/dataprepper/plugins/sink/opensearch/bulk/BulkActions.java

```diff
@@ -0,0 +1,13 @@
+package com.amazon.dataprepper.plugins.sink.opensearch.bulk;
+
+public enum BulkActions {
+
+    CREATE,
```

Comment:
If I'm reading this correctly, the user will configure the option like: `action: CREATE` in the YAML. In Data Prepper we are aiming for lower case to keep the options consistent.

You can see how we did this in the `CompressionOption` class: https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/s3-source/src/main/java/com/amazon/dataprepper/plugins/source/configuration/CompressionOption.java

And it has a corresponding unit test: https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/s3-source/src/test/java/com/amazon/dataprepper/plugins/source/configuration/CompressionOptionTest.java

As it affects the user experience, I think it is quite important.